<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plotWindow</class>
 <widget class="QMainWindow" name="plotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addPlotsButton">
        <property name="text">
         <string>Add Plots</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>37</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionPlot_tips"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionPulse_max">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pulse peak</string>
   </property>
  </action>
  <action name="actionBaseline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Baseline</string>
   </property>
  </action>
  <action name="actionSave_plot">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save plot</string>
   </property>
   <property name="toolTip">
    <string>Save plot as an image file</string>
   </property>
  </action>
  <action name="actionSave_settings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save settings...</string>
   </property>
  </action>
  <action name="actionLoad_settings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load settings...</string>
   </property>
  </action>
  <action name="actionPulse_RMS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pulse RMS</string>
   </property>
  </action>
  <action name="actionRaw_pulse_records">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Raw pulse records</string>
   </property>
  </action>
  <action name="actionTime_derivatives">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Time derivatives</string>
   </property>
  </action>
  <action name="actionErr_vs_FB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Err vs FB</string>
   </property>
  </action>
  <action name="actionFFT_sqrt_PSD">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FFT = sqrt(PSD)</string>
   </property>
  </action>
  <action name="actionNoise_PSD">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Noise PSD</string>
   </property>
  </action>
  <action name="actionAnalysis_vs_time">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Analysis vs time</string>
   </property>
  </action>
  <action name="actionAnalysis_histogram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Analysis histograms</string>
   </property>
  </action>
  <action name="actionShow_edit_ranges">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show/edit ranges</string>
   </property>
  </action>
  <action name="actionHide_ranges">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide ranges</string>
   </property>
  </action>
  <action name="actionY_axis_raw_units">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Y axis raw units</string>
   </property>
  </action>
  <action name="actionY_axis_phys_units">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Y axis phys units</string>
   </property>
  </action>
  <action name="actionPulse_mean">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pulse mean</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPlot_tips">
   <property name="text">
    <string>Plot tips</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>plotWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pausePressed(bool)</slot>
  <slot>xAxisLog(bool)</slot>
  <slot>yAxisLog(bool)</slot>
  <slot>clearGraphs()</slot>
  <slot>savePlot()</slot>
 </slots>
</ui>
